tarpc/tests/service_functional.rs

```diff
@@ -109,64 +109,3 @@ async fn concurrent() -> io::Result<()> {
 
     Ok(())
 }
-
-#[tarpc::service(derive_serde = false)]
-trait InMemory {
```

Comment:
Does removing these tests mean it's no longer possible to ensure tarpc works in a single threaded context?